onnx/defs/shape_inference.h

```diff
@@ -61,6 +60,9 @@ struct InferenceContext {
   virtual TypeProto* getOutputType(size_t index) = 0;
   virtual GraphInferencer* getGraphAttributeInferencer(const std::string& attribute_name) = 0;
   virtual ~InferenceContext() {}
+  /// XXX: Putting at the end of vtable in case it matters?
+  /// XXX: Do we need default implementation here?
+  virtual const SparseTensorProto* getInputSparseData(size_t index) const = 0;
```

Comment:
we don't need default implementation here...

Reply:
Do we care to preserve the order of vtable entries?

Reply:
Hmmm ... I don't know if there are users who want that level of compatibility or that ONNX guarantees that. But I guess this is safer anyway.